pkg/coderefs/coderefs.go

```diff
@@ -178,20 +178,19 @@ func (b *branch) findReferences(cmd command.Client, flags []string, ctxLines int
 	if err != nil {
 		return grepResultLines{}, err
 	}
-
 	return generateReferencesFromGrep(flags, grepResult, ctxLines, exclude), nil
 }
 
-func generateReferencesFromGrep(flags []string, grepResult [][]string, ctxLines int, exclude *regexp.Regexp) []grepResultLine {
+func generateReferencesFromGrep(flags []string, grepResult command.SearchResults, ctxLines int, exclude *regexp.Regexp) []grepResultLine {
```

Comment:
This file still refers to "grep". We should change that to just generically be "search", but I figured there's enough changes in this PR to warrant doing this at another time.